src/country/Country.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import { useState } from 'react';
+import { URLContext } from '../App';
+import { Unwrapped, WithOptionLabel } from '@jsonforms/material-renderers';
+import { ControlProps, OwnPropsOfEnum } from '@jsonforms/core';
+import { TranslateProps } from '@jsonforms/react';
+import { api } from '../api';
+const { MaterialEnumControl } = Unwrapped;
+
+export const Country = (
+  props: ControlProps & OwnPropsOfEnum & WithOptionLabel & TranslateProps
+) => {
+  const [options, setOptions] = useState<string[]>(['']);
+  const url = React.useContext(URLContext);
+
+  useEffect(() => {
+    api.getCountries(url).then((result) => {
+      setOptions(result);
+    });
+  }, [url]);
+
```

Comment:
For tutorial: list some code like this as an example for a loading spinner replacement
```
if(options.length === 1 && options[0] === ''){
  return <LoadingSpinner />
}
```